types/host.go

```diff
@@ -26,6 +26,33 @@ type Host interface {
 	Memory() (*HostMemoryInfo, error)
 }
 
+// NetworkCounters represents network stats from /proc/net
+type NetworkCounters interface {
+	NetworkCounters() (*NetworkCountersInfo, error)
+}
+
+// SNMP represents the data from /proc/net/snmp
+type SNMP struct {
+	IP      map[string]int64 `json:"ip" netstat:"Ip"`
+	ICMP    map[string]int64 `json:"icmp" netstat:"Icmp"`
+	ICMPMsg map[string]int64 `json:"icmp_msg" netstat:"IcmpMsg"`
+	TCP     map[string]int64 `json:"tcp" netstat:"Tcp"`
+	UDP     map[string]int64 `json:"Udp" netstat:"Udp"`
```

Comment:
I think the JSON tag was meant to be `udp`.

Reply:
Nice catch!